include/mqtt/endpoint.hpp

```diff
@@ -4191,10 +4191,10 @@ class endpoint : public std::enable_shared_from_this<endpoint<Mutex, LockGuard,
                         case control_packet_type::pingresp:
                         case control_packet_type::disconnect:
                             return remaining_length_ == 0;
-                        default:
+                        case control_packet_type::auth:
```

Comment:
v3_1_1 doesn't have auth. I think that the original code has no problem. 
See *1.

Reply:
My goal with adding auth to v3.1.1 is to allow the compiler to give us a warning about missing cases if the control_packet_type enum has other items added in the future.

Since we forgot to include auth in the v5 version, I thought it was a good idea to have the compiler warn us if we miss one in the future.